2022/12/12 03:32:42 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "favicon.ico". in /app/lib/yii-1.1.27/framework/web/CWebApplication.php:286
Stack trace:
#0 /app/lib/yii-1.1.27/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /app/lib/yii-1.1.27/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /app/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=https://study.yii.com/index.php?r=gii
---
2022/12/12 03:37:21 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'study.tbl_user' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `tbl_user`.
in /app/index.php (13)
2022/12/12 03:40:36 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'study.user' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `user`.
in /app/index.php (13)
2022/12/12 03:45:17 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '12-12-2022 10:45:32' for column 'created_at' at row 1. The SQL statement executed was: INSERT INTO `posts` (`title`, `content`, `created_at`, `updated_at`) VALUES (:yp0, :yp1, :yp2, :yp3).
in /app/protected/controllers/PostController.php (73)
in /app/index.php (13)
2022/12/12 03:45:17 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '12-12-2022 10:45:32' for column 'created_at' at row 1. The SQL statement executed was: INSERT INTO `posts` (`title`, `content`, `created_at`, `updated_at`) VALUES (:yp0, :yp1, :yp2, :yp3) in /app/lib/yii-1.1.27/framework/db/CDbCommand.php:358
Stack trace:
#0 /app/lib/yii-1.1.27/framework/db/ar/CActiveRecord.php(1083): CDbCommand->execute()
#1 /app/lib/yii-1.1.27/framework/db/ar/CActiveRecord.php(811): CActiveRecord->insert(NULL)
#2 /app/protected/controllers/PostController.php(73): CActiveRecord->save()
#3 /app/lib/yii-1.1.27/framework/web/actions/CInlineAction.php(49): PostController->actionCreate()
#4 /app/lib/yii-1.1.27/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /app/lib/yii-1.1.27/framework/web/filters/CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 /app/lib/yii-1.1.27/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /app/lib/yii-1.1.27/framework/web/CController.php(1148): CFilter->filter(Object(CFilterChain))
#8 /app/lib/yii-1.1.27/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /app/lib/yii-1.1.27/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 /app/lib/yii-1.1.27/framework/web/CController.php(291): CFilterChain->run()
#11 /app/lib/yii-1.1.27/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /app/lib/yii-1.1.27/framework/web/CWebApplication.php(282): CController->run('create')
#13 /app/lib/yii-1.1.27/framework/web/CWebApplication.php(141): CWebApplication->runController('post/create')
#14 /app/lib/yii-1.1.27/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /app/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php/post/create
HTTP_REFERER=https://study.yii.com/index.php/post/create
---
2022/12/12 03:46:04 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in /app/lib/yii-1.1.27/framework/web/auth/CAccessControlFilter.php:183
Stack trace:
#0 /app/lib/yii-1.1.27/framework/web/auth/CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 /app/lib/yii-1.1.27/framework/web/filters/CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 /app/lib/yii-1.1.27/framework/web/CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 /app/lib/yii-1.1.27/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 /app/lib/yii-1.1.27/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 /app/lib/yii-1.1.27/framework/web/CController.php(291): CFilterChain->run()
#6 /app/lib/yii-1.1.27/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /app/lib/yii-1.1.27/framework/web/CWebApplication.php(282): CController->run('admin')
#8 /app/lib/yii-1.1.27/framework/web/CWebApplication.php(141): CWebApplication->runController('post/admin')
#9 /app/lib/yii-1.1.27/framework/base/CApplication.php(185): CWebApplication->processRequest()
#10 /app/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/post/admin
HTTP_REFERER=https://study.yii.com/index.php/post/index
---
2022/12/12 06:10:05 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '12-12-2022 10:45:32' for column 'created_at' at row 1. The SQL statement executed was: UPDATE `posts` SET `id`=:yp0, `title`=:yp1, `content`=:yp2, `created_at`=:yp3, `updated_at`=:yp4 WHERE `posts`.`id`=3.
in /app/protected/controllers/PostController.php (97)
in /app/index.php (13)
2022/12/12 06:10:05 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '12-12-2022 10:45:32' for column 'created_at' at row 1. The SQL statement executed was: UPDATE `posts` SET `id`=:yp0, `title`=:yp1, `content`=:yp2, `created_at`=:yp3, `updated_at`=:yp4 WHERE `posts`.`id`=3 in /app/lib/yii-1.1.27/framework/db/CDbCommand.php:358
Stack trace:
#0 /app/lib/yii-1.1.27/framework/db/ar/CActiveRecord.php(1756): CDbCommand->execute()
#1 /app/lib/yii-1.1.27/framework/db/ar/CActiveRecord.php(1130): CActiveRecord->updateByPk(3, Array)
#2 /app/lib/yii-1.1.27/framework/db/ar/CActiveRecord.php(811): CActiveRecord->update(NULL)
#3 /app/protected/controllers/PostController.php(97): CActiveRecord->save()
#4 [internal function]: PostController->actionUpdate('3')
#5 /app/lib/yii-1.1.27/framework/web/actions/CAction.php(115): ReflectionMethod->invokeArgs(Object(PostController), Array)
#6 /app/lib/yii-1.1.27/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(PostController), Object(ReflectionMethod), Array)
#7 /app/lib/yii-1.1.27/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#8 /app/lib/yii-1.1.27/framework/web/filters/CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#9 /app/lib/yii-1.1.27/framework/web/filters/CFilter.php(40): CFilterChain->run()
#10 /app/lib/yii-1.1.27/framework/web/CController.php(1148): CFilter->filter(Object(CFilterChain))
#11 /app/lib/yii-1.1.27/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#12 /app/lib/yii-1.1.27/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#13 /app/lib/yii-1.1.27/framework/web/CController.php(291): CFilterChain->run()
#14 /app/lib/yii-1.1.27/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 /app/lib/yii-1.1.27/framework/web/CWebApplication.php(282): CController->run('update')
#16 /app/lib/yii-1.1.27/framework/web/CWebApplication.php(141): CWebApplication->runController('post/update')
#17 /app/lib/yii-1.1.27/framework/base/CApplication.php(185): CWebApplication->processRequest()
#18 /app/index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/index.php/post/update/3
HTTP_REFERER=https://study.yii.com/index.php/post/update/3
---
